spec091.go

```diff
@@ -1174,7 +1234,7 @@ func (msg *queueDeclare) id() (uint16, uint16) {
 }
 
 func (msg *queueDeclare) wait() bool {
-	return !msg.NoWait
+	return true && !msg.NoWait
```

Comment:
This is odd 🤔

Reply:
This auto-generated statement seems to be coming from here:

https://github.com/rabbitmq/amqp091-go/blob/5d0fed7eb1e3f49bd1b5967e3cd857a3591c73dc/spec/gen.go#L197-L199

It looks like the original author, Sean, did this on purpose. I agree it looks odd. I guess I can manually edit this statement?

Reply:
Ohh I forgot this file is auto-generated! That makes sense.